lib/builder/builder.js

```diff
@@ -347,6 +348,7 @@ module.exports = {
 					},
 					tasks: projectTasks,
 					project,
+					isRootProject,
```

Comment:
Alternatively we could add a reference of the root project to the BuildContext. Then you can check whether the project currently being built is equal to the root project in the build context.

Reply:
👍